files/en-us/learn/javascript/asynchronous/promises/index.html

```diff
@@ -279,7 +279,7 @@ <h3 id="Chaining_the_blocks_together">Chaining the blocks together</h3>
 
 <h2 id="Promise_terminology_recap">Promise terminology recap</h2>
 
-<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a <a href="/en-US/docs/Learn/JavaScript/Asynchronous/Promises#promise_terminology_recap">short guide that you can bookmark</a> and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
+<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a short guide that you can bookmark and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
```

Comment:
Why did you remove this link?

Reply:
The link is the current page on which it is and it's also linking to the same section on which it is, so I removed it.

Reply:
```suggestion
<p>There was a lot to cover in the above section, so let's go back over it quickly to give you a <a href="#promise_terminology_recap">short guide that you can bookmark</a> and use to refresh your memory in the future. You should also go over the above section again a few more times to make sure these concepts stick.</p>
```

What about linking to the fragment?

Reply:
I guess the link is not that necessary here.

Reply:
Oh yes, I missed this. You're right.